rest/changes_api_test.go

```diff
@@ -1709,14 +1709,17 @@ func TestOneShotChangesWithExplicitDocIds(t *testing.T) {
 	assertStatus(t, rt.SendAdminRequest("PUT", "/db/docC", `{"channels":["beta"]}`), 201)
 	assertStatus(t, rt.SendAdminRequest("PUT", "/db/docD", `{"channels":["beta"]}`), 201)
 
+	require.NoError(t, rt.WaitForPendingChanges())
+
 	// Create struct to hold changes response
 	var changes struct {
 		Results []db.ChangeEntry
 	}
 
 	//User has access to single channel
 	body := `{"filter":"_doc_ids", "doc_ids":["doc4", "doc1", "docA", "b0gus"]}`
-	request, _ := http.NewRequest("POST", "/db/_changes", bytes.NewBufferString(body))
+	request, err := http.NewRequest("POST", "/db/_changes", bytes.NewBufferString(body))
+	require.NoError(t, err)
```

Comment:
Is this actually a likely source of runtime errors, assuming the test passed at dev time?  What's the scenario where this could be failing?

Reply:
I have no idea why I made this change. I agree that runtime-dependent errors are not going to happen.